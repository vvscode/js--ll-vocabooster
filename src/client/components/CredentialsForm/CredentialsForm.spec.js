import React from 'react';
import { shallow } from 'enzyme';
import Component from './CredentialsForm';

describe('CredentialsForm', () => {
  let props;
  let getShallowComponent;

  beforeEach(() => {
    props = {};

    getShallowComponent = () => shallow(<Component {...props} />);
  });

  describe('render', () => {
    it('should render with all props', () => {
      expect(getShallowComponent()).toMatchSnapshot();
    });
  });
});
