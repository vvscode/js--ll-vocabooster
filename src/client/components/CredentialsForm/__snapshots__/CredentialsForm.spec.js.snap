// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CredentialsForm render should render with all props 1`] = `
<Form
  as="form"
>
  <FormGroup
    widths="equal"
  >
    <FormField
      control={[Function]}
      name="login"
      onChange={[Function]}
      placeholder="Lingualeo Login"
    />
    <FormField
      control={[Function]}
      name="password"
      onChange={[Function]}
      placeholder="Password"
      type="password"
    />
    <FormField
      control={[Function]}
    >
      Submit
    </FormField>
  </FormGroup>
  <pre>
    {}
  </pre>
</Form>
`;
